src/Core/Content/Media/Subscriber/MediaDeletionSubscriber.php

```diff
@@ -139,9 +135,9 @@ private function handleMediaDeletion(array $affected, Context $context): void
             }
 
             if ($mediaEntity->isPrivate()) {
-                $privatePaths[] = $this->urlGenerator->getRelativeMediaUrl($mediaEntity);
+                $privatePaths[] = $mediaEntity->getPath();
```

Comment:
what when path is empty? You need I guess an MediaLoadedSubscriber to fill path if it is empty

Reply:
yes, next todo

Reply:
implemented